src/senselab/audio/tasks/speech_enhancement/api.py

```diff
@@ -0,0 +1,36 @@
+"""This module provides the API for the senselab speech enhancement task."""
+from typing import List, Optional
+
+import pydra
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.audio.tasks.speech_enhancement.speechbrain import SpeechBrainEnhancer
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import HFModel, SenselabModel
+
+
+def enhance_audios(
+    audios: List[Audio], 
+    model: SenselabModel = HFModel(path_or_uri="speechbrain/sepformer-wham16k-enhancement", revision="main"), 
```

Comment:
should this be SenselabModel or just HFModel? Do we think there'll be a point where other models will be able to work?